openshift/osd-network-verifier
# Osd 11484 change to generic error type

All error types are currently showing as EgressError.

Changed this to generic ONV error. Added unit tests for both error types.

To test locally:
1. make build
2. Disallow all inbound traffic to your subnet 
3.  ./osd-network-verifier egress --subnet-id your_subnet  --region your_region

should return:
 "network verifier error: internet connectivity problem: please ensure there's internet access in given vpc subnets"

Egress errors for specific endpoints should still show as egress URL error. 

4. To run tests `make test`
 
 Jira https://issues.redhat.com/browse/OSD-11484
